openff/toolkit/topology/molecule.py

```diff
@@ -1089,17 +1089,20 @@ def has_unique_atom_names(self) -> bool:
         """``True`` if the molecule has unique atom names, ``False`` otherwise."""
         return _has_unique_atom_names(self)
 
-    def generate_unique_atom_names(self):
+    def generate_unique_atom_names(self, suffix: str = None):
```

Comment:
```suggestion
    def generate_unique_atom_names(self, suffix: str = None):
```
I think it'd be simpler to have the default be the current behavior. This also has the benefit of this argument always being a string

Reply:
The default is set in _generate_unique_atom_names to "x", but I guess consistency in the calls will avoid a slip.